definitions/node.d.ts

```diff
@@ -1,21 +1,29 @@
-// Type definitions for Node.js v0.10.1
+// Type definitions for Node.js v0.12.0
```

Comment:
Updated - there were previously two copies of this definition, and both were different versions.